Documentation/minio-object-store-crd.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Minio Object Store CRD
+weight: 40
+indent: true
+---
+
+# Minio Object Store CRD
+Minio object stores can be created and configured using the `objectstores.minio.rook.io` custom resource definition (CRD). Complete instructions can be found in the [Rook Minio Documentation](minio-object-store.md).
+
+##Sample
```

Comment:
I think a space is needed between the `##` and `Sample` to get it to render as a header. https://github.com/tonya11en/rook/blob/5c143ff48aac049d120c49b6f63859243622d47a/Documentation/minio-object-store-crd.md is not rendering OK.